proxy/proxy.go

```diff
@@ -34,29 +35,32 @@ const (
 	HeaderGroups = "x-pomerium-authenticated-user-groups"
 )
 
+var (
```

Comment:
should these be const?

Reply:
Yes, unfortunately Go doesn't support const structs (AFAIK). 

I split the difference and made the constant values strings and then cast them to URLs in the constructor body. Let me know what you think.

Reply:
Ah.  Yeah that makes sense.